### PR TITLE
Fixes sub-second sleep waiting for process end (#73)

### PR DESCRIPTION
**Pull request by blakev**
Addresses #73 by simplifying multi-process shutdown logic.

### CODE REVIEW

**Comment by santagada** [comment]
But how do you stop sanic now? You removed the signal handlers and ignores ctrl-c.

**Comment by blakev** [comment]
Because the event propagates to the forked processes, you'd have to issue it twice otherwise. When you call `Event.wait(..)` it's waiting for the internal flag to be set, which doesn't really matter if we're shutting the server down anyway. All shutdown functions are handled in the event loop prior to the process halting.

If you run it and try to sigkill it looks like this, 

```
(sanic-dev)blake@bwv-desktop:~/code/github/sanic$ python simple_server.py 
2016-10-19 16:48:14,215: INFO: Goin' Fast @ http://0.0.0.0:8000
2016-10-19 16:48:14,215: INFO: Spinning up 4 workers...
2016-10-19 16:48:14,217: INFO: Starting sanic-server-0, pid: 10509
2016-10-19 16:48:14,218: INFO: Starting sanic-server-1, pid: 10510
2016-10-19 16:48:14,218: INFO: Starting sanic-server-2, pid: 10511
2016-10-19 16:48:14,219: INFO: Starting sanic-server-3, pid: 10512
2016-10-19 16:48:16,020: INFO: Spinning down workers...
2016-10-19 16:48:16,021: INFO: Stop requested, draining connections...
2016-10-19 16:48:16,021: INFO: Stop requested, draining connections...
2016-10-19 16:48:16,021: INFO: Stop requested, draining connections...
2016-10-19 16:48:16,021: INFO: Stop requested, draining connections...
2016-10-19 16:48:16,022: INFO: Server Stopped
```

File: sanic/http.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+# >>
+#     Copyright (c) 2016, Blake VandeMerwe
```
Comment (rzwiefel) [inline sanic/http.py]: Did you mean to leave in this preamble?

Comment (blakev) [inline sanic/http.py]: nah, I gotta redo and squash this pull request -- too many formatting errors and a sneaky file got in there somehow from the cookies branch :)

**Comment by seemethere** [comment]
I think this was fixed in later revisions of the multiprocess serve function.

Will reopen if necessary.

**Pull request closed without merging**